### DISCUSSION TITLE
posts/hugo-stack/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#45] [Comments]
# posts/hugo-stack/

博客文章显示「最后修改于 ……」可以帮助读者了解文章的最新状况，也让自己掌握一篇文章的成熟度。
[&hellip;] 这个博客由 Hugo 生成，目前用的是 Stack 主题，里边的文件配置是可以有文章页面下方的「最后更新于 ……」功能的，摸索了好久都没成功，今天终于在 这篇博客 的帮助下搞出来了，记录一下操作。
[&hellip;] 在 config.yaml 文件中添加以下设置（后续还会修改这 &hellip;

https://shitao5.org/posts/hugo-stack/

**Comment by wastu01** [comment]
謝謝提供解決方式

**Comment by Gecko042** [comment]
谢谢您！真的写的非常好！

**Reply by Gecko042** [reply]
还是想问您一下  尝试修改文章时间一致的这个part，按照您给的配置修改后本地和部署后的网页文章最后都不显示lastmod信息了，这一点应该怎么修改呢？

**Reply by Shitao5** [reply]
您好，这个主题我已经切换掉了，如果按照文章以及[参考文章](https://centurio.net/2022/01/24/set-last-modified-date-automatically/)的操作都没奏效，可能要考虑是否是作者更新了主题导致的。

**Reply by Gecko042** [reply]
好的  非常感谢您！
